docs/quickstarts/chrome-extension.mdx

```diff
@@ -0,0 +1,259 @@
+---
+title: Chrome Extension Quickstart
+description: Add authentication and user management to your Chrome Extension with Clerk.
+---
+
+# Chrome Extension Quickstart
+
+<TutorialHero
+  framework="nextjs"
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk application",
+      link: "https://clerk.com/docs/quickstarts/setup-clerk",
+      icon: "clerk",
+    },
+    {
+      title: "Create a React application with Vite",
+      link: "https://vitejs.dev/guide/#scaffolding-your-first-vite-project",
+      icon: "nextjs",
+    },
+    {
+      title: "Install the CRXJS Plugin",
+      link: "https://crxjs.dev/vite-plugin/getting-started/react/create-project",
+      icon: "nextjs",
+    }
+  ]}
+>
+
+- Install `@clerk/chrome-extension`
+- Set your environment variables
+- Add `<ClerkProvider>` to your application
+- Create a header with Clerk components for users to sign in and out
+- Add basic styles
+
+</TutorialHero>
+
+This tutorial only supports the email & password, email & email code, and SMS & SMS code [authentication options](/docs/authentication/configuration/sign-up-sign-in-options). Email & email link, social sign in (OAuth), SAML, and invites are not supported.
+
+<Steps>
+
+### Install `@clerk/chrome-extension`
+
+{/* TODO: Fill the link the reference */}
+Clerk's [Chrome Extension SDK](#) gives you access to prebuilt components, React hooks, and helpers to make user authentication easier.
+
+Add the SDK to your project:
+
+<CodeBlockTabs type="installer" options={["npm", "yarn",  "pnpm" ]}>
+  ```bash filename="terminal"
+  npm install @clerk/chrome-extension
+  ```
+  ```bash filename="terminal"
+  yarn add @clerk/chrome-extension
+  ```
+
+  ```bash filename="terminal"
+  pnpm add @clerk/chrome-extension
+  ```
+</CodeBlockTabs>
+
+### Set your environment variables
+
+<SignedIn>
+
+Add the following keys to your `.env.local` file. These keys can always be retrieved from the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page of your Clerk Dashboard.
```

Comment:
```suggestion
Add the following keys to your `.env` file. These keys can always be retrieved from the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page of your Clerk Dashboard.
```

`.env.local` is supported in Vite but `.env` seems to be the preferred method by the community.